contrib/azure/azure-coreos-cluster

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+
+from azure import *
+from azure.servicemanagement import *
+import argparse
+import urllib2
+import time
+import base64
+import os
+import subprocess
+
+parser = argparse.ArgumentParser(description='Create a CoreOS cluster on Microsoft Azure.')
+parser.add_argument('--version', action='version', version='azure-coreos-cluster 0.1')
```

Comment:
I think we should use `0.1` as the return value to follow suit with our other clients:

```
$ deis --version
1.4.0-dev
$ deisctl --version
1.4.0-dev
```